tests/functional/test_freeze.py

```diff
@@ -134,23 +122,14 @@ def test_freeze_git_clone(script, tmpdir):
         """
             Script result: ...pip freeze
             -- stdout: --------------------
-            ...-e %s@...#egg=pip_test_package-...
+            ...-e git+...#egg=version_pkg-master
```

Comment:
I'd almost be tempted to say that ```_create_test_package``` should take the name to use as an argument (defaulting to the current value). Explicit is better than implicit and all that :-) But I guess a comment is sufficient for this PR.